extensions/mssql/src/utils.ts

```diff
@@ -68,7 +68,7 @@ export function getConfigTracingLevel(): string {
 }
 
 export function getDefaultLogDir(): string {
-	return path.join(process.env['VSCODE_LOGS'], '..', '..', 'mssql');
+	return path.join(process.env['ADS_LOGS'], '..', '..', 'mssql');
```

Comment:
doesn't vscode expose a api for logging now?

Reply:
After looking into it, yes, they expose a log path as part of the context the extension gets passed on activation. not that big of a deal for this pr